src/components/skeleton/Skeleton.module.css

```diff
@@ -0,0 +1,43 @@
+.skeleton {
+  --loader-background-color: #eeeeee;
+  --loader-highlight-color: #dedede;
```

Comment:
I've noticed that these two colors stand apart from the color scheme we are going with :( is it possible to pick colors which will work out well in both light/dark mode? you can define them in the variables.css file. 

if it's okay, there is a --skeleton variable defined in that file, you can increase/decrease the opacity to make it look the way you want :D

Reply:
I would like to move this color to to variable scss file so that all the color schemes remain in the same file. Although it's a small thing, but later we might not look into that. 
But overall animation looks good, I also like the option of type where user can select the shape of their choice 💯